CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,16 @@
+# Marked code of coduct
+
+## Values
+
+- ...
+
+## Principles
+
+- ...
+
+## Practices
+
+- **Final doesn't exist:** It is assumed that each of us is working to the best of their ability to make Marked better using the knowledge and skills we each have at the time.
+- **Perfection is the unattainable goal:** The long we wait to make something perfect, the longer our users have to wait for the features and functionality the want and need to be delighted with Marked. Having said that, we will always strive to continuously improve in all areass.
```

Comment:
The `long` or the `longer` we wait?

Reply:
`areass` should probably be `areas`
Maybe run this through a spell check first.

Reply:
lol - Fair. Was mainly throwing it up here to see what y'all wanted to add; or, if y'all thought these were not worth having. Updated title to be more explicit of where the PR is, sorry about that.

Reply:
I think it would be easiest to grab an existing code of conduct and remove anything we don't agree with. That's what GitHub recommends:

> Wherever you can, use prior art. The Contributor Covenant is a drop-in code of conduct that is used by over 40,000 open source projects, including Kubernetes, Rails, and Swift.
> The Django Code of Conduct and the Citizen Code of Conduct are also two good code of conduct examples.

Reply:
I saw that recommendation as well - and think it may be a good one. 

Under normal circumstances, I would facilitate a gathering where we actually talked about this stuff - good team building exercise - it just so happens to wind up with cultural norms at the end. 

Thought this might be a good alternative and give us a chance to actually build something together as a team, which I suppose removing things might cover as well.

Which one do you think is a better for us to start with??

Reply:
Which of the two mentioned in the quote...?? 

(I also tend toward the 8fold one, which is why I picked the format, I prefer it over legalize or dry one, you know? Don't want it to be milquetoast, of course, I guess that's really up to the contributors in the end.)

Anyway...probably not focused enough on it to really take in all the potential impacts before feeling comfortable taking action.

https://8fold.pro/about

Reply:
The 8fold about page doesn't look like a code of conduct.
How about the node.js [CODE_OF_CONDUCT.md](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md)

After all, this is a project that runs on node.

Reply:
Yeah, the 8fold one emphasizes the concept of culture all around as opposed to "thou shalt do this and not do that". But, it could be odd for people accustomed to the other type. I do appreciate the NodeJS one and don't have anything that I would fundamentally disagree with.